comixed-webui/src/app/library/components/comic-covers/comic-covers.component.ts

```diff
@@ -302,7 +304,15 @@ export class ComicCoversComponent implements OnInit, OnDestroy, AfterViewInit {
     return this.readComicIds.includes(comic.id);
   }
 
-  private sortData(): void {
+  downloadComicData(comics: Comic[]): void {
+    this.logger.debug('Downloading comic metadata:', comics);
+    this.fileDownloadService.saveFileContent({
+      content: new Blob([JSON.stringify(comics)], { type: 'application/json' }),
+      filename: 'test-metadata.json'
```

Comment:
Yes, it would. I wasn't sure what would be a good name or name-generation method, so just threw that in. I'll change it to your suggestion and update the PR.